atat_provisioning_wizard_api.yaml

```diff
@@ -1054,8 +1066,10 @@ components:
           pattern: '^[a-zA-Z\d _-]{4,100}$'
         description:
           type: string
+          pattern: ^[\w\d !@#$%^&*_|:;,'.-]{0,300}$
```

Comment:
Business Rules say:
> Alphanumeric (A-z, 0-9) + space + special characters (interpreted as !@#$%^&*_|:;,'.-)
> Max length: 300 characters
👍

❔ The description is optional.  Is it still possible to omit a description?